### DISCUSSION TITLE
empty output

### DISCUSSION THREAD

**Discussion by kangahdesmond** [giswqs/geog-312#135] [Q&A] [answered]
![question](https://github.com/user-attachments/assets/a1e5fab8-c316-4141-a5b0-282eef8cdb5b)
![question](https://github.com/user-attachments/assets/f7552c16-ae87-44ce-870b-90708675d24a)

when ever i print like image.shape an empty [] comes without the values appearing

**Comment by giswqs** [comment, marked as answer]
Make sure the vector and raster datasets are in the same crs.
